### PR TITLE
Fix logout URL for France Connect V2

### PR DESCRIPTION
**Pull request by micedre**
Fix url

### CODE REVIEW

File: src/main/java/fr/insee/keycloak/providers/common/AbstractBaseIdentityProvider.java [hunk 1]
```diff
@@ -95,6 +97,10 @@ public Response keycloakInitiatedBrowserLogout(KeycloakSession session, UserSess
         .build();
   }
 
+  public abstract String getIdTokenForLogout(UserSessionModel userSession);
```
Comment (mboisnard) [inline src/main/java/fr/insee/keycloak/providers/common/AbstractBaseIdentityProvider.java:100]: I think you can have the default behavior here:

```
protected String getIdTokenForLogout(UserSessionModel userSession) {
     return userSession.getNote(FEDERATED_ID_TOKEN);
}
```

File: src/main/java/fr/insee/keycloak/providers/franceconnect/FranceConnectIdentityProvider.java [hunk 2]
```diff
@@ -72,10 +73,18 @@ protected UriBuilder createAuthorizationUrl(AuthenticationRequest request) {
...
-
+    
     return validateToken(token, ignoreAudience);
   }
 
+
+  
+  @Override
+  public String getIdTokenForLogout(UserSessionModel userSession) {
+      var idToken = userSession.getNote(FEDERATED_ID_TOKEN);
```
Comment (mboisnard) [inline src/main/java/fr/insee/keycloak/providers/franceconnect/FranceConnectIdentityProvider.java:84]: And `var idToken = super(userSession);` here.

**Review by mboisnard — CHANGES_REQUESTED** [review]

**Comment by micedre** [comment]
you're right, thanks. I'll wait for an answer from FC on the reason to pass decrypted token here before marking that PR as ready again.

**Comment by micedre** [comment]
No response from FC, I'll merge this, to make a new release

**Pull request merged**